smoke-test-pipeline.yml

```diff
@@ -103,7 +103,7 @@ stages:
         itemPattern: "spk*"
         artifactName: 'spk_linux_node_12'
         targetPath: '$(System.DefaultWorkingDirectory)'
-    - bash: |in 
```

Comment:
How did this work?

Reply:
This is a compile error that was added when the variable group was switched